tests/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+#
+# Development configuration of dependent services for Docker Compose.
+#
+postgresql:
+    image: postgres:9.4
+    environment:
+        POSTGRES_USER: resolwe
+        POSTGRES_DB: resolwe
+    ports:
+        - "55432:5432"
```

Comment:
I suggest we change the port to `55431` to avoid the clash with docker-compose configuration from [resolwe-bio](https://github.com/genialis/resolwe-bio/blob/master/tests/docker-compose.yml).